lib/sablon/operations.rb

```diff
@@ -50,6 +50,13 @@ def evaluate(context)
         block.replace []
       end
     end
+
+    class Image < Struct.new(:image_reference, :block)
+      def evaluate(context)
+        image = image_reference.evaluate(context)
```

Comment:
Should we add some kind of check here to make sure `image` was evaluated to something that can replace a `ImageBlock`?